### PR TITLE
Feature/aapd 97 conservation upload

### PR DESCRIPTION
**Pull request by suityou01**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-97

## Description of change

Convervation area upload

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 1]
```diff
@@ -63,11 +63,10 @@ exports.questions = {
 		validators: [new RequiredValidator()]
 	}),
 	conservationAreaUpload: new MultiFileUploadQuestion({
-		title: 'Conservation area map and guidance',
+		title: 'Upload conservation map and guidance',
```
[the remark this answers is not in the document]
Comment (chrisHowarthkc) [reply packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:66]: This isn't actually a change in requirements, its missing from the ticket on looking, however the content on the prototype aligns with the previous content. The title is referring to the text on the questionnaire summary as Paul mentioned, where the 'title' mentioned on the ticket is the 'question' field not the title. Which I can understand being misleading

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 2]
```diff
@@ -156,19 +155,23 @@ exports.questions = {
...
+		question: '**************placeholder*********************',
+		fieldName: 'representation-upload'
+	}),
+	reportUpload: new MultiFileUploadQuestion({
+		title: '**************placeholder*********************',
+		question: '**************report-upload-placeholder*********************',
+		fieldName: 'report-upload'
 	}),
-	planningOfficersReportUpload: new MultiFileUploadQuestion({
+	planningOfficersUpload: new MultiFileUploadQuestion({
```
Comment (chrisHowarthkc) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:167]: I think this section is the artifact of a merge? the name was changed my pr merged earlier to 'planningOfficersReportUpload'. Pr for reference: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3055

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 3]
```diff
@@ -156,19 +155,23 @@ exports.questions = {
...
 	representationUpload: new MultiFileUploadQuestion({
-		title: 'Upload representations from other parties',
-		question: 'Upload the representations',
-		fieldName: 'representation-upload',
-		validators: [new RequiredValidator('You must add your documents')]
+		title: '**************placeholder*********************',
+		question: '**************placeholder*********************',
+		fieldName: 'representation-upload'
+	}),
+	reportUpload: new MultiFileUploadQuestion({
```
Comment (chrisHowarthkc) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:162]: This section was a duplicate of the below and removed in the below pr earlier today, i think this is a merge artifact: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3055

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 4]
```diff
@@ -156,19 +155,23 @@ exports.questions = {
 		validators: [new RequiredValidator()]
 	}),
 	representationUpload: new MultiFileUploadQuestion({
-		title: 'Upload representations from other parties',
-		question: 'Upload the representations',
-		fieldName: 'representation-upload',
-		validators: [new RequiredValidator('You must add your documents')]
+		title: '**************placeholder*********************',
+		question: '**************placeholder*********************',
+		fieldName: 'representation-upload'
```
Comment (chrisHowarthkc) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:160]: This content was just updated in another Pr for the representation upload ticket, think this is also a merge/rebase artifact: 
https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3054

**Review by chrishawldev — DISMISSED** [review]
Please make changes in line with feedback and the heroku prototype on the jira ticket

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 5]
```diff
@@ -65,9 +65,8 @@ exports.questions = {
 	conservationAreaUpload: new MultiFileUploadQuestion({
 		title: 'Conservation area map and guidance',
 		question: 'Upload conservation map and guidance',
-		description:
-			'<a href="https://magic.defra.gov.uk/magicmap.aspx" target="_blank" class="govuk-link">Link to Magic Maps (opens in new tab)</a>',
-		fieldName: 'conservation-area-upload'
+		fieldName: 'conservation-area-upload',
+		validators: [new RequiredValidator('You must add your documents')]
```
Comment (edpackard-work) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:69]: for multifile validation, this line should now read:
`validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]`

**Review by edpackard-work — COMMENTED** [review]
looks like something has gone awry in the rebasing? it might be quicker to start a new branch with the actual changes you made

**Review by chrisHowarthkc — APPROVED** [review]

**Comment by suityou01** [comment]
> looks like something has gone awry in the rebasing? it might be quicker to start a new branch with the actual changes you made

I don't know what you mean. Can you please be more specific? What has done awry?

**Pull request merged**